os-eap64-launch/added/launch/tx-datasource.sh

```diff
@@ -67,7 +67,11 @@ function generate_tx_datasource() {
 }
 
 function inject_jdbc_store() {
-  jdbcStore="<jdbc-store datasource-jndi-name=\"${1}ObjectStore\"/>"
+  jdbcStore="<jdbc-store datasource-jndi-name=\"${1}\">\\
+                <action table-prefix=\"\${jboss.node.name}\"/>\\
```

Comment:
This causes an SQL syntax error (at least) on Postgres

2018-01-04 13:48:17 UTC txobj 10.131.0.1(55534) 2/10 0 ERROR:  syntax error at or near "-" at character 18
2018-01-04 13:48:17 UTC txobj 10.131.0.1(55534) 2/10 0 STATEMENT:  CREATE TABLE node-1JBossTSTxTable (StateType INTEGER NOT NULL, Hidden INTEGER NOT NULL, TypeName VARCHAR(255) NOT NULL, UidString VARCHAR(255) N
OT NULL, ObjectState bytea, PRIMARY KEY(UidString, TypeName, StateType))